### PR TITLE
Sunw3 add a what to do file for Nashua

### LINKED ISSUE #17: Add a file for sunw3 hometown
# Adding file for Nashua, NH

- [x]  Create a branch

- [x] Add a file

- [x] Commit my changes

- [x]  open a pull request

- [x]  Get a code review

- [ ]  get approval

- [ ]  Merge the PR:)

### PR DESCRIPTION
**Pull request by sunw3**
This PR resolve #17 

@githubteacher what can you do when in Nashua NH

### CODE REVIEW

File: Nashua.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Nashua, NH
+
```
Comment (misstonbon) [inline Nashua.txt]: Never been there, gonna Google it!

Comment (sunw3) [inline Nashua.txt]: in northeast. Close to ski places:)

Comment (misstonbon) [inline Nashua.txt]: Oh, I wish I didn't mind  snow. I much prefer the sun, which is kind of strange since I live in Seattle.

Comment (sunw3) [inline Nashua.txt]: Being at Portland for 6 months 10+ years ago. Like the fist few months when it was sunny and warm. But then rainy season started and no sun for weeks:(. Good city for all kinds of outdoor activities, though.

Thanks for reviewing:)

**Pull request merged**